rollbar-java/src/test/java/com/rollbar/notifier/util/ObjectsUtilsTest.java

```diff
@@ -0,0 +1,55 @@
+package com.rollbar.notifier.util;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.Arrays;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class ObjectsUtilsTest {
+  private static final Object[] EQUALS_VALUES = {"123", 123, null};
+  private static final String[] HASH_VALUES = {"one", "two", "three"};
+  private static String TEST = "Test";
+  private static String ERROR_DESCRIPTION = "Error description";
+	
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+	
+  @Test
+  public void testEquals() {
```

Comment:
I think this test would be better using the JUnit Parameterized runner.

I don't know if it's the diff in github but it seems there is some weird indentation in the whole file.